TLM/TLM/Manager/Impl/SpeedLimitManager.cs

```diff
@@ -420,25 +404,19 @@ public class SpeedLimitManager : AbstractGeometryObservingManager, ICustomDataMa
 				return -1;
 			}
 
-			/*if (!(info.m_netAI is RoadBaseAI))
-				return -1;*/
-
-			//string infoName = ((RoadBaseAI)info.m_netAI).m_info.name;
-			string infoName = info.name;
-			int speedLimitIndex;
-			if (!CustomLaneSpeedLimitIndexByNetInfoName.TryGetValue(infoName, out speedLimitIndex)) {
-				return AvailableSpeedLimits.IndexOf(GetVanillaNetInfoSpeedLimit(info, true));
-			}
-
-			return speedLimitIndex;
+			var infoName = info.name;
+			float speedLimit;
+			return !CustomLaneSpeedLimitByNetInfoName.TryGetValue(infoName, out speedLimit)
```

Comment:
For long statements like these, I personally prefer seeing the brackets instead of a one-line to improve readability.

Reply:
Scratch this one. If the one-line were to be made into one-line then it'd work better.